lib/webpacker/source.rb

```diff
@@ -29,4 +31,8 @@ def digested_filename
     def filename
       "#{name}.js"
     end
+
+    def dev_server?
+      Rails.configuration.x.webpacker[:dev_server] && Rails.configuration.x.webpacker[:dev_server] != false
+    end
```

Comment:
Instead of this, I'd like to add an accessor method like this:

```ruby
def config
  Rails.configuration.x.webpacker
end
```

Which we can then use like:

```ruby
def path
  if config[:dev_host].present?
    "#{config[:dev_host]}/#{filename}"
  elsif digesting?

  ...
end
```

That'll follow the pattern from `asset_host` we already use.